src/router/index.ts

```diff
@@ -1,13 +1,17 @@
 import { createRouter, createWebHistory } from "vue-router/auto";
 import index from "../pages/index.vue";
 import about from "../pages/about.vue";
+import accessibility from "../pages/accessibility.vue";
+import privacy from "../pages/privacy.vue";
 import notFound from "../pages/notFound.vue";
 
 const router = createRouter({
     history: createWebHistory(import.meta.env.BASE_URL),
     routes: [
         { path: "/:pathogen?/:version?/:indicator?/:country?", component: index, props: true },
-        { path: `/about`, component: about },
+        { path: "/:pathogen?/about", component: about },
+        { path: "/:pathogen?/accessibility", component: accessibility },
+        { path: "/:pathogen?/privacy", component: privacy },
```

Comment:
I think these should use the supported `PATHOGEN` defined in router utils for the moment, otherwise they happily show up if you go to e.g. `/malaria/about` - eventually (when we have multiple pathogens) we should abstract out the route checking that the index page does, but I think we can just hardcode it for now.